### PR TITLE
Added Cond 

### LINKED ISSUE #17525: cond

### PR DESCRIPTION
**Pull request by avi-thakur-0812**
Close #17525

### CODE REVIEW

**Comment by avi-thakur-0812** [comment]
@zaeemansari70  Hey! Could you please review my PR?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py [hunk 1]
```diff
@@ -17,6 +17,34 @@
 )
 
 
+#cond
+@handle_frontend_test(
+    fn_tree="jax.numpy.linalg.cond",
+    dtype_and_x=_get_dtype_and_matrix_non_singular(),
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py:23]: `_get_dtype_and_matrix_non_singular()`, is not defined and implemented anywhere.

File: ivy/functional/backends/torch/linear_algebra.py [hunk 2]
```diff
@@ -509,10 +509,6 @@ def vector_norm(
 vector_norm.support_native_out = True
 
 
-# Extra #
```
Comment (zaeemansari70) [inline ivy/functional/backends/torch/linear_algebra.py:512]: Kindly un-commit your change from this file, you do not need to remove this.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hi @avi-thakur-0812 ,

I've requested some changes below :slightly_smiling_face: 

Do let me knwo of you have any other questions!
Thanks!

**Pull request closed without merging**